src/assets/database.sql

```diff
@@ -358,6 +387,6 @@ INSERT INTO `tools` (`id`, `name`, `fullname`, `categories`, `description`, `sit
 INSERT INTO `tools` (`id`, `name`, `fullname`, `categories`, `description`, `site`, `github`, `released`, `avatar`, `cmd`, `target`, `resume`, `category`, `category2`, `solution`) VALUES (24, 'h2t', 'HTTP Hardening Tool', 'benign vulnerability-analysis', 'h2t (HTTP Hardening Tool) scans a website and suggests security headers to apply', 'https://certbahia.pop-ba.rnp.br/projects/h2t/', 'https://github.com/gildasio/h2t', 'Yes', NULL, 'h2t scan', '-- ', NULL, 'Benign', 'Vulnerability Analysis', NULL);
 INSERT INTO `tools` (`id`, `name`, `fullname`, `categories`, `description`, `site`, `github`, `released`, `avatar`, `cmd`, `target`, `resume`, `category`, `category2`, `solution`) VALUES (25, 'arjun', 'arjun', 'information-gathering web-application', 'HTTP parameter discovery suite', NULL, 'https://github.com/s0md3v/Arjun', 'Yes', 'assets/img/arjun.png', 'arjun', '-u', NULL, 'Information-Gathering', 'Web-Application', NULL);
 
-
+INSERT INTO `tools` (`id`, `name`, `fullname`, `categories`, `description`, `site`, `github`, `released`, `avatar`, `cmd`, `target`, `resume`, `category`, `category2`, `solution`) VALUES (27, 'dnsenum', 'tool for DNS enumeration to find DNS Servers', 'DNS enumeration', 'The main purpose of Dnsenum is to gather as much information as possible about a domain.', 'https://www.kali.org/tools/dnsenum/', 'https://github.com/fwaeytens/dnsenum', 'No', 'assets/img/dnsenum.png', 'dnsenum', NULL, 'DNS Enumeration', 'Vulnerability Analysis', 'Information Gathering')
```

Comment:
I think `fullname` and `categories` are mixed a little bit. `fullname` could be `DNS enumeration` and not the short description.